mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/316 - camera follow character

Closes #316 

To test:
- Run Training/Life with Macros or Starting Scenarios/Patches scenario
- Make sure camera follows character while game is unpaused
- Make sure moving the camera manually (panning, zooming, rotating, or moving to next viewpoint/building/user/etc) pauses the game
